### ISSUE TITLE
Warning: EOS.undent is deprecated

### ISSUE THREAD

**Issue by Jmuccigr** [OSGeo/homebrew-osgeo4mac#324]
I get these warnings when I do `brew info gdal2`:

```
Warning: Calling <<-EOS.undent is deprecated!
Use <<~EOS instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2.rb:310:in `caveats'
Please report this to the osgeo/osgeo4mac tap!

Warning: Calling <<-EOS.undent is deprecated!
Use <<~EOS instead.
/usr/local/Homebrew/Library/Taps/osgeo/homebrew-osgeo4mac/Formula/gdal2.rb:310:in `caveats'
Please report this to the osgeo/osgeo4mac tap!
```

**Comment by nickrobison** [comment]
Thanks for reporting, I believe this has been resolved in #308.

**Comment by Jmuccigr** [comment]
Yep, seems ok in 2.2.4.
